compiler/src/org.graalvm.compiler.lir/src/org/graalvm/compiler/lir/VariableMap.java

```diff
@@ -87,4 +85,38 @@ public void filter(Predicate<T> predicate) {
         }
     }
 
+    public boolean isEmpty() {
+        if (content.isEmpty()) {
+            return true;
+        }
+
+        boolean empty = true;
+        for (T e : content) {
+            if (e != null) {
+                empty = false;
+                break;
+            }
+        }
+        return empty;
```

Comment:
I'd rewrite this to 
```java
        for (T e : content) {
            if (e != null) {
                return false;
            }
        }
        return true;
```
Not a blocker, though.

Reply:
Good catch! Thanks!